example-get-started/generate.sh

```diff
@@ -331,8 +342,8 @@ EOF
   dvc repro train
   tick
   git commit -am "${COMMIT_PREFIX}Reproduce model using bigrams"
-  git tag -a "9-bigrams-model${GIT_TAG_SUFFIX}" -m "Model retrained using bigrams."
-  if [ $OPT_REGISTER_MODELS == "true" ]; then
+  create_tag "9-bigrams-model${GIT_TAG_SUFFIX}" "Model retrained using bigrams."
+  if [ $OPT_TAG_MODELS == "true" ]; then
     gto register "${GTO_PREFIX}text-classification" --version v1.1.0
```

Comment:
Shouldn't this be also updated to use `OPT_MODEL_NAME` instead of `text-classification`?

Reply:
that's a really good catch! thanks a lot @daavoo 🙏  I'll update it and will update the tags in the new repo I generated with it (I'm hitting some issues on BE that @amritghimire has handled already, after that I'll create PR with a migration to this new fixture)